pontoon/contributors/templates/contributors/settings.html

```diff
@@ -11,161 +11,314 @@
-            {{ Checkbox.checkbox('Project target dates', class='project-deadline-notifications', attribute='project_deadline_notifications', is_enabled=user.profile.project_deadline_notifications, title='Get notified when project target date approaches') }}
-            {{ Checkbox.checkbox('Comments', class='comment-notifications', attribute='comment_notifications', is_enabled=user.profile.comment_notifications, title='Get notified when comments are submitted to your strings') }}
-            {{ Checkbox.checkbox('New suggestions ready for review', class='unreviewed-suggestion-notifications', attribute='unreviewed_suggestion_notifications', is_enabled=user.profile.unreviewed_suggestion_notifications, title='Get notified when new suggestions are ready for review') }}
-            {{ Checkbox.checkbox('Review actions on own suggestions', class='review-notifications', attribute='review_notifications', is_enabled=user.profile.review_notifications, title='Get notified when your suggestions are approved or rejected') }}
-            {{ Checkbox.checkbox('New team contributors', class='new-contributor-notifications', attribute='new_contributor_notifications', is_enabled=user.profile.new_contributor_notifications, title='Get notified when new contributors make their first contributions to the team') }}
-        </ul>
-    </section>
-
-    <section>
-        <h3>Editor</h3>
-        <ul class="check-list">
-            {{ Checkbox.checkbox('Translate Toolkit checks', class='quality-checks', attribute='quality_checks', is_enabled=user.profile.quality_checks, title='Run Translate Toolkit checks before submitting translations') }}
-
-            {% if user.can_translate_locales %}
-                {{ Checkbox.checkbox('Make suggestions', class='force-suggestions', attribute='force_suggestions', is_enabled=user.profile.force_suggestions, title='Save suggestions instead of translations') }}
-            {% endif %}
-        </ul>
-    </section>
-
-    <section>
-        <h3>Default locales</h3>
-        <div id="locale-settings" class="clearfix">
-            <div id="homepage">
-                <span class="label">Homepage</span>
-                {{ team_selector.locale(locales, locale) }}
-            </div>
-            <div id="preferred-locale">
-                <span class="label">Preferred source locale</span>
-                {{ team_selector.locale(preferred_locales, preferred_locale) }}
+                {% else %}
+                <p class="help">If you would like to provide a different email address from your login email for email
+                    communications, please do so here. Note, this email will also appear under your <a
+                        href="{{ url('pontoon.contributors.profile') }}">Profile page</a>.</p>
+                {% endif %}
+            </div>
+            <div class="check-list">
+                {{ Checkbox.checkbox(
+                'News and updates',
```

Comment:
@bcolsson With the introduction of the "Monthly activity summary" checkbox, "Email communications" seems like a too broad term (because an activity summary is also sent via email, so it's a form of email communication). See the screenshot below.

@flod suggested we change it to "Updates and News". (I flipped the order, because to me it sounds more natural that way, perhaps because "news" is a broader term and "updates" feels more specific.)

Note: We'll also need to update the value of EMAIL_COMMUNICATIONS_HELP_TEXT on prod, because it refers to "email communications".

<img width="709" alt="Screenshot 2024-11-29 at 15 14 35" src="https://github.com/user-attachments/assets/16507495-42c3-42f1-96f1-0f88c24780dc">

Reply:
I'm okay with updating to `News and Updates` but we should adjust some of the other text to make sure it's clear that everything is an email (despite the header being Email, I think the descriptions should also include that.)

Reply:
`EMAIL_COMMUNICATIONS_HELP_TEXT = Get emails about the latest updates about localization at Mozilla, announcements about new Pontoon features, invitations to contributor events and more.<br/><br/>By enabling News and Updates emails, I agree to Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`

Reply:
Thanks!

Maybe replace the first "about" with "with" to avoid repetition?

`EMAIL_COMMUNICATIONS_HELP_TEXT = Get emails with the latest updates about localization at Mozilla, announcements about new Pontoon features, invitations to contributor events and more.<br/><br/>By enabling News and Updates emails, I agree to Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`